languages/go/exercises/concept/time-duration/.docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+## Time
+
+In Go, functionality for working with times is provided by the `time` package. The types and methods in this package allow us to manipulate times, get the current time, determine elapsed time, parse times from strings, and more.
+
+To work with time, you will usually call a method on a `Time` instance, but there are also some functions called on the `time` package itself.
+
+Parsing strings works a little differently in Go. Many languages use a format string with various codes like YYYY for four-digit year, MM for two-digit month, etc. Go instead uses an exact date, `Mon Jan 2 15:04:05 -0700 MST 2006`, to parse strings. You can rearrange these components or omit them as necessary, and you can spell out month and day names, or use their number values, but in order for Go to understand the parts of the string you are parsing, the corresponding parts of the format string must be from this exact date.
```

Comment:
I was slightly confused by "Go instead uses an exact date". What do you mean by that? Reading the entire section, I interpreted this as: "Go doesn't allow for specifying a format string". Is that right?

Reply:
Go uses the 2nd Jan, 2006 15:04:05 -0700 as reference date as the format. So parsing a date string looks like this:

```go
t, err := time.Parse("2006-06-23", "2020-08-15")
```

The first "date" is the format. Usually a format is put into a constant anyway so it does not look that weird:

```go
t, err := time.Parse(dateFormat, "2020-08-15")
```

Reply:
Ah right! I didn't quite get that from the text. It might be me :)